Tensile/Tests/pre_checkin/preload_kernel_arguments_always_half.yaml

```diff
@@ -0,0 +1,56 @@
+TestParameters:
+  marks: [xfail-gfx900, xfail-gfx906, xfail-gfx908,
```

Comment:
Will this work on the following arch in the future?
I would like to make sure why you use "xfail" here instead of "skip".

Reply:
No they won't.  I suppose we could just use skip here.